styles/components/_search.scss

```diff
@@ -141,6 +141,14 @@
       padding: 0px;
     }
   }
+
+  .o-acp__item {
+    padding: 0;
+    @include ktheme() {
```

Comment:
why include ktheme if you don't use `theme` ?

Reply:
cc @prachi00